### DISCUSSION TITLE
Private profiles are treated like new users

### DISCUSSION THREAD

**Discussion by Craftplacer** [community/community#34787] [Other Feature Feedback, Questions, & Ideas]
When you set your profile to private, your contribution graph shows a notice as if you were a new user. This seems like an internal bug.

![image](https://user-images.githubusercontent.com/22963120/193403367-92d55d54-0cc3-4d4e-94cc-3d1b7dcc198a.png)

*Also the send feedback link links to a category (`profile-feedback`) that doesn't exist...*

**Comment by radar-blip** [comment]
You can file a bug report if you like, here: [Bug-Report](https://support.github.com/contact/bug-report)
